features/ACM/upgrade_risks.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Upgrade risks
```

Comment:
@falox Can you add here a note stating that this kind of requests are going to be run on demand. ACM team confirmed it [in this message](https://redhat-internal.slack.com/archives/C05E6S43CGP/p1692880725097519?thread_ts=1692880562.268419&cid=C05E6S43CGP) and we would like to have it stated here as it's quite relevant for our agreement with RHOBS. Thanks!

Reply:
well, actually it's implicit in each scenario. Never mind